clusters/build-clusters/common/00_namespaces.yaml

```diff
@@ -6,3 +6,13 @@ metadata:
       organization
     openshift.io/display-name: OpenShift CI
   name: ci
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ci-stg
```

Comment:
We decided to not use this namespace anymore and ppl just use `pj-rehearse` so we can just remove it

Reply:
Removed.